### PR TITLE
Enabled frozen string literals

### PR DESCRIPTION
**Pull request by shopmike**
@dylanahsmith @pushrax @fw42 @ashmaroli

### CODE REVIEW

File: lib/liquid/block_body.rb [hunk 1]
```diff
@@ -113,7 +115,7 @@ def blank?
     end
 
     def render(context)
-      render_to_output_buffer(context, '')
+      render_to_output_buffer(context, +'')
```
Comment (shopmike) [inline lib/liquid/block_body.rb]: I'm not sure if this is the best approach. This means frozen strings will fail if passed in to `render_to_output_buffer`. The other option is to allow frozen strings to be passed in and `render_to_output_buffer` does the unfreezing

Comment (shopmike) [reply lib/liquid/block_body.rb]: I guess this is equivalent to what it is now.

File: .rubocop_todo.yml [hunk 2]
```diff
@@ -83,14 +83,7 @@ Style/ClassVars:
...
     - 'test/unit/context_unit_test.rb'
-
-# Offense count: 119
-# Cop supports --auto-correct.
-# Configuration parameters: EnforcedStyle.
-# SupportedStyles: always, never
-Style/FrozenStringLiteralComment:
-  Enabled: false
-
+    -
```
Comment (ashmaroli) [inline .rubocop_todo.yml]: ??

File: lib/liquid/block_body.rb (hunk 1 above)
Comment (ashmaroli) [reply lib/liquid/block_body.rb]: Yes. The API is clear. This expects an unfrozen string. If a frozen string is sent anyways, then a `FrozenError` exception will be called with the caller at fault.

File: test/integration/drop_test.rb [hunk 3]
```diff
@@ -31,7 +33,7 @@ def text
 
   class CatchallDrop < Liquid::Drop
     def liquid_method_missing(method)
-      'catchall_method: ' << method.to_s
+      +'catchall_method: ' << method.to_s
```
Comment (dylanahsmith) [inline test/integration/drop_test.rb]:
```suggestion
      "catchall_method: #{method}"
```

**Pull request merged**